src/System.Linq/src/System/Linq/Aggregate.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
We don't have a policy written down, but personally I don't think this is any different from any other using, which means most code in src in corefx doesn't need 'em due to being in the System namespace.